src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -248,9 +250,29 @@ private void setupKeyboardShortcuts() {
                     break;
                 }
             }
+            if (SHOW_RELATED_ISSUE_OR_PR.match(event) && ui.getBrowserComponent().isCurrentUrlIssue()) {
+                if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
+                    showRelatedIssueOrPR();
+                } else { // only for default. can remove if default key for MARK_AS_READ_CHANGES
```

Comment:
You might want to check the key for mark as read here also. If it's changed and we press E, this will still trigger.